lib/game.rb

```diff
@@ -0,0 +1,75 @@
+class Game
+
+  def initialize
+    @total_score = 0
+    @strike = false
+    @spare = false
+    @current_frame = 1
+  end
+
+
+  def play_frame
+    puts "Whats is the score of your first bowl?"
+    bowl_1 = gets.chomp.to_i
+      if bowl_1 == 10
+        bowl_2 = 0
+      else
+        puts "Whats is the score of your second bowl?"
+        bowl_2 = gets.chomp.to_i
+      end
+    frame_score = calculate_frame_score(bowl_1, bowl_2)
+    @total_score += frame_score
+    p "Your current score is #{@total_score}, not including pending bonus points ;)"
+    # below evaluates whether the current frame is a strike/spare/neither,
+    # so it changes the variable value for the next frame to adjust its scores accordingly
+    strike?(bowl_1)
+    spare?(bowl_1, bowl_2)
+  end
+
+  # def bonus_round
+  #   if current_frame
+  # end
+
+  def calculate_frame_score(bowl_1, bowl_2)
+    if @strike == true
+      #this adds the
+      score = (bowl_1 + bowl_2) * 2
+    elsif @spare == true
+      score = (bowl_1 * 2) + bowl_2
+    elsif
+      score = bowl_1 + bowl_2
+    end
+    return score
+  end
+
+
+  def strike?(bowl_1)
+    if bowl_1 == 10
+      @strike = true
+    else
+      @strike = false
+    end
+  end
+
+  def spare?(bowl_1, bowl_2)
```

Comment:
Could use a single line boolean here to condense code. Something like this:

roll1 != 10 && roll1 + roll2 == 10 ? @spare = true : @spare = false